src/pgrepr-consts/src/oid.rs

```diff
@@ -292,10 +292,10 @@ pub const FUNC_IS_RBAC_ENABLED_OID: u32 = 16_574;
 pub const FUNC_MZ_ACL_ITEM_CONTAINS_PRIVILEGE_OID: u32 = 16_575;
 pub const FUNC_MZ_VALIDATE_PRIVILEGES_OID: u32 = 16_576;
 pub const FUNC_ROLE_OID_OID: u32 = 16_577;
-pub const FUNC_MZ_NAME_TO_GLOBAL_ID_FULL_QUAL: u32 = 16_578;
-pub const FUNC_MZ_NAME_TO_GLOBAL_ID_ANY_SCHEMA: u32 = 16_579;
-pub const FUNC_MZ_NAME_TO_GLOBAL_ID_ONE_SCHEMA: u32 = 16_580;
-pub const FUNC_MZ_NAME_TO_GLOBAL_ID_ITEM_NAME: u32 = 16_581;
+pub const FUNC_MZ_RESOLVE_OBJECT_NAME_FULL: u32 = 16_578;
+pub const _FUNC_MZ_NAME_TO_GLOBAL_ID_ANY_SCHEMA: u32 = 16_579;
+pub const _FUNC_MZ_NAME_TO_GLOBAL_ID_ONE_SCHEMA: u32 = 16_580;
+pub const FUNC_MZ_RESOLVE_OBJECT_NAME: u32 = 16_581;
```

Comment:
Is there any concern with re-purposing OIDs? I don't think we write them down anywhere so I think it should be fine.

Reply:
Yeah, for internal functions like this I didn't think it mattered because the only place where they change would be the mz_objects table which could/would only be used by e.g. ORMs/drivers, none of which would be looking for this function.